packages/jazz-nodejs/src/index.ts

```diff
@@ -41,31 +40,66 @@ export async function startWorker<Acc extends Account>({
     throw new Error("Invalid accountSecret");
   }
 
-  const { account: worker, done } = await createJazzContext({
+  const context = await createJazzContext({
     auth: fixedCredentialsAuth({
       accountID: accountID as ID<Acc>,
       secret: accountSecret as AgentSecret,
     }),
     AccountSchema,
     // TODO: locked sessions similar to browser
     sessionProvider: randomSessionProvider,
-    peersToLoadFrom: [wsPeer],
+    peersToLoadFrom: [wsPeer.peer],
     crypto: await WasmCrypto.create(),
   });
 
-  setInterval(async () => {
```

Comment:
This was preventing the scripts from ending because it keeps a timer active even when the work is "done"